client/web/reporter/src/views/Report/Builder.vue

```diff
@@ -511,6 +515,8 @@ export default {
 
         selected: undefined,
       },
+
+      gridKey: 0,
```

Comment:
You dont need the gridKey if you change the v-if for the grid
That would rerender it

Reply:
Resetting the reportBlocks should be theoretically enough right? I tested it and all it does for a split second is show the "no blocks" message before rendering correctly. Should I just reset the reportBlocks and not add the condition, since that is one less logic?

Reply:
Nope, you need processing spinner if report is fetching/processing